src/components/Column/ColumnBody.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { render, screen, within, fireEvent, act } from "@testing-library/react";
+import { mockGetComputedStyle, mockDndSpacing } from "react-beautiful-dnd-test-utils";
+
+import Ticket from "../../interfaces/Ticket";
+import { ColumnBody, DraggableColumnBody } from "./ColumnBody";
+
+describe("<ColumnBody />", () => {
+  test("passed appropriate props - It should render given ticket with title", () => {
+    // Arrange
+    const firstTicketTitle = mockedTickets[0].name;
+    const secondTicketTitle = mockedTickets[1].name;
+    const thirdTicketTitle = mockedTickets[2].name;
+    const mockedOnEditCallback = jest.fn();
+
+    render(<ColumnBody tickets={mockedTickets} onEditClick={mockedOnEditCallback} />);
+
+    // Act
+    screen.getAllByTestId("ticketEditButton").forEach((element) => {
+      fireEvent.click(element);
+    });
+
+    // Assert
+    expect(screen.getByText(firstTicketTitle)).toBeTruthy();
+    expect(screen.getByText(secondTicketTitle)).toBeTruthy();
+    expect(screen.getByText(thirdTicketTitle)).toBeTruthy();
+    expect(mockedOnEditCallback).toBeCalledTimes(mockedTickets.length);
+  });
+});
+
+describe("<DraggableColumnBody />", () => {
+  beforeEach(() => {
+    mockGetComputedStyle();
+  });
+
+  test("passed appropriate props - It should render given ticket with title", async () => {
+    // Arrange
+    const mockedColumnTitle = "Lorem Ipsum";
+    const firstTicketTitle = mockedTickets[0].name;
+    const secondTicketTitle = mockedTickets[1].name;
+    const thirdTicketTitle = mockedTickets[2].name;
+
+    const mockedSetTicketCallback = jest.fn();
+    const mockedOnEditCallback = jest.fn();
+
+    const { container } = render(
+      <DraggableColumnBody
+        title={mockedColumnTitle}
+        tickets={mockedTickets}
+        setTickets={mockedSetTicketCallback}
+        onEditClick={mockedOnEditCallback}
+      />,
+    );
+    mockDndSpacing(container);
+
+    // Act
+    screen.getAllByTestId("ticketEditButton").forEach((element) => {
+      fireEvent.click(element);
+    });
+    // this function doesn't run drag on test. Need further research.
+    // await makeDnd({
+    //   getDragElement: () => screen.getByText(firstTicketTitle).closest(DND_DRAGGABLE_DATA_ATTR),
+    //   direction: DND_DIRECTION_DOWN,
+    //   positions: 2,
+    // });
```

Comment:
Sorry I was trying to implement a test for DnD part, but the function doesn't grab the element found by getDragElement. I'm not sure if it's because of our implementation or the library itself. I wasted hours on this damn problem and eventually gave up.